resources/config/snemo/demonstrator/simulation/geant4_control/2.3/README.rst

```diff
@@ -0,0 +1,178 @@
+================================================================
+Geant4 simulation for SuperNEMO demonstrator
+================================================================
+
+:Authors: François Mauger, Yves Lemière
+:Date:    2018-01-19
+
+.. contents::
+   :depth: 3
+..
+
+Presentation
+============
+
+This directory  contains the files  needed to run Geant4 simulation
+for the SuperNEMO demonstrator simulation.
+
+* Version is : ``2.2``
+
+This setup reuses a large part of the  ``2.1`` setup depends on:
+
+* the SuperNEMO demonstrator's geometry layout (``4.0``) and its variants.
+* the SuperNEMO vertex generation (``4.1``) and its variants.
+* the SuperNEMO primary event generation (``1.3``) and its variants.
+
+Files
+=====
+
+  * ``README.rst`` : This file.
+  * ``manager.conf``  :  The main  configuration  file for the
+  Bayeux/mctools Geant4 simulation engine.
+  * ``variants/``  : This directory contains configuration files
+    for variant support.
+
+Check the configuration
+=======================
+
+First make sure the Bayeux and Falaise softwares are installed and setup
+from a (Cadfael)brew shell:
+
+.. code:: sh
+
+   brewsh> which bxvariant_inspector
+   ...
+   brewsh> which bxg4_production
+   ...
+..
+
+Browse and edit primary event generation variant parameters and options
+===============================================================================
+
+1. From the build directory, generate documentation about variants:
+
+.. code:: sh
+
+   brewsh> bxvariant_inspector \
+          --datatools::resource-path="falaise@$(pwd)/BuildProducts/share/Falaise-3.0.0/resources" \
+          --variant-config "@falaise:config/snemo/demonstrator/simulation/geant4_control/2.2/variants/repository.conf" \
```

Comment:
The versions/tags here don't match current ones (3.2.0 for Falaise, 2.3 for this tag).